internal/models/interfaceconfig.go

```diff
@@ -0,0 +1,78 @@
+package models
+
+import (
+	"bytes"
+	"text/template"
+)
+
+var (
+	temp           = template.Must(template.New("Tunnel").Parse(tunnelTemplate))
```

Comment:
Why did this get renamed `temp`?   Its a template, so calling it `tmpl` was fine.